apps/web-giddh/src/environments/environment.local.electron.ts

```diff
@@ -7,6 +7,7 @@ export const environment: Environment = {
     showDevModule: true,
     AppUrl: 'http://localhost:4200/',
     ApiUrl: 'https://api.giddh.com/',
+    DevApiUrl: 'http://giddh-api-prod-g.eu-west-2.elasticbeanstalk.com/',
```

Comment:
use https url

Reply:
As confirmed with Ashish and Kriti, we are using this URL. However, if it's for HTTPS, we don't have a certificate for it.

Reply:
@dilpreetsinghofficial will this work on test?